setup.cfg

```diff
@@ -11,7 +11,6 @@ addopts =
     --tb native
     -W ignore
     --cov=alibi_detect
-    --cov-append
```

Comment:
Sorry was going to add a comment about this once the ci had finished etc.

The default for `cov-append` is false. My understanding is that it is intended to be used in scenarios like if tests are split across multiple `tox` envs, so you need to combine the results of those separate tests at the end. I don't think it really matters for CI because each build is run on a totally isolated system anyway. But when running locally I was thinking that `--cov-append` might cause confusion unless the user is careful to clear the coverage files manually after changes are made? Was going to check this one with you though as wasn't sure if there was a reason the flag was added...

Reply:
Thinking back I might have added it because I think I was getting an error trying to run tests locally a 2nd time without removing the coverage files manually.

Reply:
I didn't think of that. I shall run some tests to see if that is still a problem.

Reply:
Conclusion is I *think* it is OK to remove. Running tests multiple times seems to be OK. However, we could also just leave it in to be safe... The CI builds are isolated so leaving it in should cause any jumbled-up coverage reports AFAIK.